packages/gatsby-plugin-image/src/components/static-image.server.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FunctionComponent } from "react"
+import { splitProps, AllProps } from "../utils"
+import { FluidObject, FixedObject } from "gatsby-image"
+import { GatsbyImage as GatsbyImageServer } from "./compat"
```

Comment:
I assume we're moving away from the compat version as next steps?

Reply:
Yes